packages/react-examples/src/react-components/FluentUIComponents/StylingComponents.stories.mdx

```diff
@@ -0,0 +1,120 @@
+import { Meta, Source } from '@storybook/addon-docs/blocks';
+
+<Meta title="Concepts/Styling Components" />
+
+## Styling components
+
+To style Fluent UI components `makeStyles` is used.
+
+`makeStyles` is a proprietary CSS-in-JS implementation which generates atomic CSS classes and can be imported from "@fluentui/react-components".
```

Comment:
how is `makeStyles` proprietary ?, do you mean 'our custom implementation' ?

Reply:
I mean it's not 3rd party OSS library

Reply:
Let's settle with smth this, since proprietary seems to me that we'll get the lawyers involved if you fork it :)

> homegrown CSS-in-JS implementation (inspired by stylex?)